networkx/algorithms/bipartite/tests/test_cluster.py

```diff
@@ -59,6 +59,7 @@ def test_average_path_graph():
     assert bipartite.average_clustering(G, mode="dot") == 0.5
     assert bipartite.average_clustering(G, mode="max") == 0.5
     assert bipartite.average_clustering(G, mode="min") == 1
+    assert bipartite.average_clustering(G, nodes=[0, 2], mode="min") == 1
```

Comment:
Hmm, is there any list of nodes you could pass in for this particular graph where the end result *wouldn't* be 1? It might be better to develop a test case where the expected output of `average_clustering` for a subset of nodes is actually different than the value for the entire graph.